about.html

```diff
@@ -0,0 +1,189 @@
+                    </p>
+                </div>
+        
+            </div>
+
+            <div class="row pb-5 align-items-center text-center justify-content-center">
+                <div class=" col-md-4 summit-logo p-2">
+                   <img id="summit" src="assets/images/logo/Tech.png" alt="techlife logo" class="img-fluid">
+                </div> 
+            </div>
+        </div>
+        <hr>
+    </section>
+
+    <!-- feature speakers -->
+    <section id="past-summit">
+        <div class="container-lg">
+            <div class="text-center">
+                <h2 class="summit-heading pt-4 pb-2">See the Past TechLife Global Summits</h2>
+                <div class="line"></div>
+                <p class="past-p p-3">Take a look at the past two TechLife Global Summits which took place in Herat and in Kabul </p>
+            </div>
+            <div class="row align-items-center justify-content-center pb-5 pt-2 g-1">
+
+                <div class="col-md-4 me-2 text-center">
+                    <div class="img-fluid">
+                        <div class="img-container">
+                            <img src="assets/background/past-summit-1.png" alt="Techlife 2021 summit" class="img-fluid">
+
+                            <div class="content pt-5">
+                              <h2 class="img-header">2021</h2>
+                              <p class="img-text">Techlife Innovative Global Summit in 2021</p>
+                            </div>
+                        </div>
+                    </div>              
+                </div>  
+
+                <div class="col-md-4 me-2 text-center">
+                    <div class="img-fluid">
+                        <div class="img-container">
+                            <img src="assets/background/past-summit-2.png" alt="Techlife 2021 summit" class="img-fluid">
+
+                            <div class="content pt-5">
+                              <h2 class="img-header">2022</h2>
+                              <p class="img-text">Techlife Innovative Global Summit in 2022</p>
+                            </div>
+                        </div>
+                    </div>              
+                </div>  
+
+                
+                 
+            </div>
+        </div>
+    </section>
+
+    <!-- footer  -->
+
+    <section id="footer">
+        <div class="container-fluid about-footer py-5">
+                <div class="col-md-8 offset-md-2">
+                    <div class="footer-p-logo">
+                        <div class="footer-logo">
+                            <img src="assets/images/logo/techlife.png" alt="mozilla" class="img-fluid logo">
+                        </div>
+                        <p class="about-footer-p footer-p">2023 Creative Commons Korea. Some Rights Reserved.</p>
+                    </div>
+                </div> 
+        </div>
+    </section>
```

Comment:
- [ ] To correctly match the template design, kindly ensure the **Partner** section is present in the Mobile version of your About page and the footer is supposed to have a dark background color. Below is the image of the template design
![Screenshot from 2023-07-06 19-28-04](https://github.com/MozamelJawad/Capstone-Project/assets/83042742/d24bb03c-4817-40ac-b097-a83421d72e02)